python/test/options.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# options.py
+
+import argparse
+
+import os
+
+"""
+parse_arguments
```

Comment:
This documentation string isn't very clear to me.
The namespace argument is not used, so remove it.
You probably don't need any of the documentation, because the function itself is very clear.

Reply:
Done.